src/Components/ScheduleList/stubs/schedule-list.php

```diff
@@ -0,0 +1,8 @@
+<?php
```

Comment:
Why we need this configuration file?

Reply:
The ``hmazter/laravel-schedule-list`` package contains a config for strings to remove from the command name - I have edited the default to include the application name. If we didn't have this the command name would start with 'application' (or whatever the app name is) and this would have a knock on effect so the description doesn't work.